cocos/core/data/garbage-collection.ts

```diff
@@ -81,4 +81,7 @@ class GarbageCollectionManager {
 }
 
 const garbageCollectionManager = new GarbageCollectionManager();
+/**
+ * @engineInternal
+ */
 export { garbageCollectionManager };
```

Comment:
```suggestion
/**
 * @engineInternal
 */
const garbageCollectionManager = new GarbageCollectionManager();
export { garbageCollectionManager };
```

Reply:
It's better to add document comment on variable declaration expression, not to add on export expression.